metricbeat/module/kubernetes/state_container/state_container.go

```diff
@@ -138,9 +138,20 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 	m.enricher.Enrich(events)
 
 	for _, event := range events {
+		containerFields := mapstr.M{}
+
+		// Add container.name field (same as kubernetes.container.name)
+		if containerName, ok := event["name"]; ok {
+			cName, ok := (containerName).(string)
+			if !ok {
+				m.Logger().Debugf("Error while casting containerImage: %s", ok)
```

Comment:
```suggestion
				m.Logger().Debugf("Error while casting containerName: %s", containerName)
```
here should be `containerName`, isn't it?